safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -86,14 +92,24 @@ def __init__(self, parent=None, iface=None):
         self.parent = parent
         self.setupUi(self)
 
-        self.setWindowTitle(self.tr('PetaJakarta Downloader'))
+        self.setWindowTitle(self.tr('PetaBencana Downloader'))
 
         self.iface = iface
 
+        self.source = None
+
+        self.buttonGroup.setExclusive(True)
+        self.radioButton_prod.setChecked(True)
```

Comment:
Please be sure that these radio buttons are hidden when we are not in developer mode.